CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Moved coverage and performance data from GitHub gist to coreax-metadata repo.
   (https://github.com/gchq/coreax/pull/887)
+- Equinox dependency version is changed from `<0.11.8` to `>=0.11.9`. (https://github.com/gchq/coreax/pull/898)
```

Comment:
This is technically a breaking change, and we should highlight it as such!

Reply:
Is there a special format for this? Also, the contributor guidelines state that we need a deprecation period for breaking changes, but it seems like overkill in this instance.

Reply:
I don't think there's a specific format for it; [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) doesn't specify one. I'd just add `**[BREAKING CHANGE]**` to the start of the line to make it obvious.

I agree that a deprecation period seems overkill here, especially since there's not really a way to have a transition period.

However, I'm not sure we actually need to limit the Equinox version so tightly in the first place - the tests seem to run fine with Equinox versions older than `0.11.9` (e.g. `0.11.7`)? We should definitely have _some_ lower bound on the Equinox version though - e.g. I'd imagine Coreax won't work with `equinox==0.0.1`!

Reply:
From my very rigorous testing, the tests fail for 0.11.0 but pass for 0.11.5, so the latter should be relatively safe. A potential issue is that the dev environment will often resolve to the latest version so we won't really know if something breaks for the older versions.

Reply:
In which case, the latter is a good lower bound. Since tests fail for earlier versions, I think bumping the minimum required version isn't a breaking change _per se_, since it's just making a pre-existing requirement explicit, but it's still worth highlighting in the changelog.

Checking our lower bounds are correct is something that we can do by setting uv's [`--resolution-strategy`](https://docs.astral.sh/uv/concepts/resolution/#resolution-strategy) to `lowest` (or possibly `lowest-direct` in a pinch) - I'm planning on adding this in a CI job at some point, either as part of resolving #807 or as a subsequent PR.